### PR TITLE
Fix: Loader icon is stuck

### PR DESCRIPTION
**Pull request by banklesss**

### CODE REVIEW

File: src/hooks/index.ts [hunk 1]
```diff
@@ -702,3 +704,18 @@ export const useLogout = () => {
     dispatch(signout())
   }
 }
+
+export const useRefreshOnFocusHack = () => {
```
Comment (wolverineks) [inline src/hooks/index.ts]: move this out of here
this is for cross platform hooks only

**Review by wolverineks — CHANGES_REQUESTED** [review]
see comments

File: src/hooks/index.ts (hunk 1 above)
Comment (banklesss) [reply src/hooks/index.ts]: done

**Review by stackchain — APPROVED** [review]
WR PR.

File: src/TxHistory/TxHistoryList/TxHistoryList.tsx [hunk 2]
```diff
@@ -58,6 +58,27 @@ export const TxHistoryList = ({onScrollUp, onScrollDown, ...props}: Props) => {
...
 }
 
+// workaround for https://emurgo.atlassian.net/browse/YOMO-199
+// related to https://github.com/facebook/react-native/issues/15694
+export const useRefreshOnFocusHack = () => {
+  const [key, setKey] = React.useState(0)
+  const navigation = useNavigation()
+
+  React.useEffect(() => {
+    if (Platform.OS !== 'ios') {
```
Comment (wolverineks) [inline src/TxHistory/TxHistoryList/TxHistoryList.tsx:68]: why add this?

Comment (banklesss) [reply src/TxHistory/TxHistoryList/TxHistoryList.tsx:68]: Discussed on call

**Pull request merged**